docs/rules/curly/index.html

```diff
@@ -12,12 +12,47 @@
   In the code above, the author almost certainly meant for both `foo++` and `bar++`
   to be executed only if `foo === bar`. However, he forgot braces and `bar++` will be executed
   no matter what. This rule could prevent such a mistake.
-optionsDescription: Not configurable.
-options: null
+optionsDescription: |
+
+  The rule may be set to `true`, or to any combination of these arguments:
+
+  * `"all"` requires braces for all control-flow statements
+  * `"same-line-single-line"` requires braces for control-flow statements
```

Comment:
my issue with these settings is that it doesn't consider that there may be a rule in the future that requires that the brace *not* be there. All the names here assume that they all combine together to a set of places where braces are required.